ethereum/core/src/main/java/org/hyperledger/besu/ethereum/mainnet/MainnetTransactionProcessor.java

```diff
@@ -339,12 +339,11 @@ public TransactionProcessingResult processTransaction(
           gasCalculator.accessListGasCost(accessListEntries.size(), accessListStorageCount);
       final long gasAvailable = transaction.getGasLimit() - intrinsicGas - accessListGas;
       LOG.trace(
-          "Gas available for execution {} = {} - {} - {} - {} (limit - intrinsic - accessList - data)",
+          "Gas available for execution {} = {} - {} - {} (limit - intrinsic - accessList)",
           gasAvailable,
           transaction.getGasLimit(),
           intrinsicGas,
-          accessListGas,
-          blobGas);
```

Comment:
did you intend for this to be part of the bytes48 pr ?

Reply:
it is a little log fix, that I included to avoid creating a separate PR just for it